dashboard.py

```diff
@@ -63,17 +63,36 @@ def make_timeseries(originalname, df, spec) -> altair.Chart | None:
     df = df.fillna({"ci_run_url": "https://github.com/coiled/coiled-runtime"})
     path = df.path.iloc[0]
     kwargs = {}
+    # Reduce the size of the altair spec
+    df = df[
+        [
+            spec.field,
+            "start",
+            "ci_run_url",
+            "name",
+            "call_outcome",
+            "coiled_runtime_version",
+            "dask_version",
+        ]
+    ]
     if len(df.name.unique()) > 1:
         kwargs["color"] = altair.Color("name:N")
+    if len(df.call_outcome.unique()) > 1:
+        kwargs["shape"] = altair.Shape(
+            "call_outcome:N",
+            scale=altair.Scale(domain=["passed", "failed"], range=["circle", "cross"]),
```

Comment:
Good question! We should probably filter those out all-together.

Reply:
>  straightforward to just change the color of the X?

I tried to do that and found it not so straightforward. It may be possible, but I time-boxed it.

Reply:
Done